test/Microsoft.AspNet.Diagnostics.Tests/ElmPageTest.cs

```diff
@@ -0,0 +1,210 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using Microsoft.AspNet.FileProviders;
+using Xunit;
+
+namespace Microsoft.AspNet.Diagnostics
+{
+    public class ElmPageTest
+    {
+        [Theory]
+        [InlineData("TestFiles/SourceFile.txt")]
```

Comment:
Hu? It's already semi-unclear that the data to the absolute path method is relative paths. A member will at least abstract that part away.